roles/common/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+# All containers need to be up in order for this task to pass
+- name: "[TEST] verify status of multiple containers"
+  ansible.builtin.include_tasks:
+    file: container_status.yml
+  loop: "{{ containers }}"
+  loop_control:
+    label: "{{ item }}"
+  vars:
+    container_name: "{{ item }}"
+
+---
+- name: "[TEST] Verify service for logging"
```

Comment:
There should be a conditional here, so that the service_tests.yml will only be included when the right vars are defined